### PR TITLE
Refunds V2 web component integration

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7489

### Change description ###
Refunds V2 web component integration

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Mo-Lala84 — DISMISSED** [review]
Approved

**Review by Kartheek-S — DISMISSED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

File: package.json [hunk 1]
```diff
@@ -74,8 +75,8 @@
     "@angular/platform-browser-dynamic": "^11.2.14",
     "@angular/router": "^11.2.14",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "6.13.10-b",
```
Comment (deanbullock-hmcts) [inline package.json:77]: Why have we gone backwards in the version for Toolkit?

**Review by deanbullock-hmcts — COMMENTED** [review]
Just one question regarding toolkit versioning

**Review by GaryScottHMCTS — APPROVED** [review]

**Review by Kartheek-S — APPROVED** [review]

**Pull request merged**